examples/import-boss/generators/import_restrict.go

```diff
@@ -266,6 +279,65 @@ func (irf importRuleFile) verifyRules(restrictionFiles []*fileFormat, f *generat
 	return nil
 }
 
+func (irf importRuleFile) verifyInverseRules(restrictionFiles []*fileFormat, f *generator.File, path string) error {
+	selectors := make([][]*regexp.Regexp, len(restrictionFiles))
```

Comment:
can you add a short comment above each block with what it does? like a one- or twoliner.